server/src/main/java/io/crate/types/BitStringType.java

```diff
@@ -101,6 +101,9 @@ public List<DataType<?>> getTypeParameters() {
 
     @Override
     public TypeSignature getTypeSignature() {
+        if (length == DEFAULT_LENGTH) {
+            return new TypeSignature(NAME);
```

Comment:
Otherwise cannot register scalar function with BIT param, throws error
`The signature type of the based data type parameters can only be: INTEGER_LITERAL_SIGNATURE`